pkg/antctl/antctl.go

```diff
@@ -102,24 +102,24 @@ var CommandList = &commandList{
 		{
 			use:     "networkpolicy",
 			aliases: []string{"networkpolicies", "netpol"},
-			short:   "Print NetworkPolicies",
-			long:    "Print NetworkPolicies in ${component}. 'namespace' is required if 'pod' is provided.",
-			example: `  Get a specific NetworkPolicy
-  $ antctl get networkpolicy np1 -n ns1 -T acnp
-  Get the list of NetworkPolicies in a Namespace
+			short:   "Print control plane NetworkPolicies",
+			long:    "Print control plane NetworkPolicies in ${component}. 'namespace' is required if 'pod' is provided.",
+			example: `  Get a specific control plane NetworkPolicy
+  $ antctl get networkpolicy 6001549b-ba63-4752-8267-30f52b4332db
+  Get the list of all control plane NetworkPolicies
+  $ antctl get networkpolicy
+  Get the control plane NetworkPolicy with a specific source (supported by agent only)
+  $ antctl get networkpolicy -S allow-http -n ns1
```

Comment:
I still like a most convenient way to get a K8s NP. I know you guys are saying antctl is to get the internal NP objects and there are multiple times, but I would assume in the majority of cases, people are just using K8s NPs, and might not remember this "-S" flag.
But unfortunately, I cannot make out a better idea. How you think about using the "name" argument as the source NP name, when namespace is provided? It might be confusing too thoug.

Reply:
I agree this is not ideal for user that use K8s policy only. 
> But unfortunately, I cannot make out a better idea. How you think about using the "name" argument as the source NP name, when namespace is provided? It might be confusing too though.

It might be inconsistent with the convention that the first positional argument of the "get" command is the name. For example, the output of antctl get networkpolicy is now:
```
NAME                                 APPLIED-TO                           RULES SOURCE                                          
5ff083ad-299e-4c10-867e-ff5f166f8599 a604dcac-3c86-5028-b0f3-1856721a19f8 2     AntreaClusterNetworkPolicy:test-cnp             
6001549b-ba63-4752-8267-30f52b4332db 766a9e51-f132-5c2f-b862-9ac68e75d77d 1     K8sNetworkPolicy:default/allow-http             
7e91a605-74fb-4ef1-96c2-406628c1fee2 c4c59cfe-9160-5de5-a85b-01a58d11963e 2     AntreaNetworkPolicy:default/access-nginx-ingress
880db7e8-fc2a-4030-aefe-09afc5f341ad 766a9e51-f132-5c2f-b862-9ac68e75d77d 1     K8sNetworkPolicy:default/allow-https            
9d47142c-8ce5-4466-a475-6083f4087786 25b6f12b-7e53-5d04-92f4-d8c64d85b21d 1     K8sNetworkPolicy:default/allow-mysql
```
The first column is NAME and it will be naturely for user to use the value from that column to further get one specific policy. If we add a "--name" argument, it might be confusing.

Reply:
What I meant is like "antctl get netpol allow-http -n ns1". I know you might think it is confusing too..

Reply:
> What I meant is like "antctl get netpol allow-http -n ns1". I know you might think it is confusing too..

For me it's indeed a little confusing unless we are replacing the name column with the source policy name as well. From implementation perspective, there would need some special code for "get networkpolicy" command as it constructs the query in a different way.
Do you think we need a discussion about this before checking it or we can discuss it and address it in another PR? I'm trying to get this patch in as soon as possible to unblock networkpolicy status and IPv6 features.

Reply:
We can merge the current PR first. Again I sincerely do not like to remember -S (even -s is better but I know it is reserved), but I have no good idea.

Reply:
Or how about using the argument for the source name, and use a --name flag for the real name? Of course we can discuss this separately and merge this PR first.

Reply:
Sure, will open an issue to summarize the discussion.